pwnlib/tubes/ssh.py

```diff
@@ -1700,7 +1702,7 @@ def libs(self, remote, directory = None):
 
         libs = self._libs_remote(remote)
 
-        remote = self.readlink('-f',remote).strip()
+        remote = context._decode(self.readlink('-f',remote).strip())
```

Comment:
This seems to be error-prone in case of strange encodings. As it is not exposed to the user anywhere, I suppose it should be left as-is.
```suggestion
        remote = self.readlink('-f',remote).strip()
```

Reply:
If you don't decode it `bash_libs = gdb.find_module_addresses('/bin/bash', shell)` can't work 🙃

Reply:
Fair point, keeping it that way. We have no new doctests here, though, which could ensure this never breaking.